src/draudiostreamfamily.cpp

```diff
@@ -132,10 +132,10 @@ int32_t DRAudioStreamFamily::calculate_volume()
   else
   {
     // master volume adjustment
-    volume = volume * (float(DRAudioEngine::get_volume()) * 0.01f);
+    volume *= (float(DRAudioEngine::get_volume()) * 0.01f);
   }
 
-  return volume * 100.f;
+  return volume;

```

Comment:
\* 100.0f?

Reply:
We previously use `int32_t` as whole numbers but `DRAudioStream` requires a floating value. Where `0.00f` is equivalent to 0% and `1.00f` is equivalent to 100%. If you had `float volume` multiplied by 100 then it would tell BASS to play its audio at 10000% of its regular volume.

Not exactly what we want unless you want users to get deaf.